care/facility/api/serializers/patient.py

```diff
@@ -472,9 +473,20 @@ def save(self, **kwargs):
                 self.instance.save()
 
 
+class PatientNotesEditSerializer(serializers.ModelSerializer):
+    id = serializers.UUIDField(source="patient_note.external_id", read_only=True)
```

Comment:
this might be conflicting because the id wont be unique, better to not have it than be confusing